net/nginx-util/src/nginx-ssl-util.cpp

```diff
@@ -0,0 +1,515 @@
+#include <thread>
+#ifdef NO_PCRE
+#include <regex>
+#else
+#include "regex-pcre.hpp"
+#endif
+#include "px5g-openssl.hpp"
+#include "nginx-util.hpp"
+
+using namespace std;
```

Comment:
This is very bad practice. Doing this will make stuff like std::bind and bind conflict. They are very different functions.

Reply:
I removed it now :-)